src/main/java/trello/api/rest/BoardAPIMethods.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package trello.api.rest;
+
+import io.restassured.response.Response;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * BoardAPIMethods class.
+ *
+ * @author Raul Choque
+ * @version 0.0.1
+ */
+public class BoardAPIMethods {
+
+    private RestClientAPI request;
+
+    /**
+     * Initializes and construct the HTTP request.
+     */
+    public BoardAPIMethods() {
+        request = new RestClientAPI(Authentication.getRequestSpecification("admin"));
+    }
+
+    /**
+     * Creates a board with a nameBoard parameter.
+     *
+     * @param nameBoard is to set the name of board.
+     * @return a response with the name board.
+     */
+    public Response createBoard(final String nameBoard) {
```

Comment:
Please add these methods to TrelloAPIMethod class

Reply:
Thank you, this wrong was fixed.